plugins/org.jkiss.dbeaver.registry/src/org/jkiss/dbeaver/registry/RegistryConstants.java

```diff
@@ -86,6 +86,7 @@ public class RegistryConstants {
     public static final String ATTR_SAMPLE_URL = "sampleURL"; //$NON-NLS-1$
     public static final String ATTR_WEB_URL = "webURL"; //$NON-NLS-1$
     public static final String ATTR_PROPERTIES_WEB_URL = "propertiesURL"; //$NON-NLS-1$
+    public static final String ATTR_DRIVER_DOCUMENTATION_URL = "driverDocumentationURL"; //$NON-NLS-1$
```

Comment:
Please rename this attribute as well. I propose using `databaseDocumentationKey` but it might be discussed/
```suggestion
    public static final String ATTR_DATABASE_DOCUMENTATION_KEY = "databaseDocumentationKey"; //$NON-NLS-1$
```